testeng/jobs/cleanupPythonCode.groovy

```diff
@@ -0,0 +1,80 @@
+
+PYTHON_VERSION = 'python3.5'
+
+DEFAULT_BRANCH = 'master'
+
+ORG = 'edx'
+
+githubUserReviewers: ['']
+
+githubTeamReviewers: ['arbi-bom']
+
+emails: ['arbi-bom@edx.org']
+
+
+job("cleanup-python-code-job"){
+
+ parameters {
+        stringParam('repoName', null, 'URL of the target repository')
+        stringParam('pythonVersion', PYTHON_VERSION, 'Version of python to use.(optional)')
+        stringParam('packages', '', 'Comma separated list of packages to install')
+        stringParam('scripts', '', 'Comma separated list of scripts to run')
+    }
+
+      environmentVariables(
+            REPO_NAME: repoName,
+            ORG: ORG,
+            PR_USER_REVIEWERS: githubUserReviewers.join(','),
+            PR_TEAM_REVIEWERS: githubTeamReviewers.join(',')
+        )
+
+        multiscm {
+            git {
+                remote {
+                    credentials('jenkins-worker')
+                    url("git@github.com:edx/${repoName}.git")
+                }
+                branch("master")
+                extensions {
+                    cleanBeforeCheckout()
+                    relativeTargetDirectory(repoName)
+                }
+            }
+            git {
+                remote {
+                    url('https://github.com/edx/testeng-ci.git')
+                }
+                branch('master')
+                extensions {
+                    cleanBeforeCheckout()
+                    relativeTargetDirectory('testeng-ci')
+                }
+            }
+        }
+
+          steps {
+           def packagesToInstall = packages.tokenize(",")
+           def scriptsToRun = scripts.tokenize(",")
```

Comment:
I suspect you may need to do something differently here to access the parameters also.